packages/console/src/pages/Users/index.tsx

```diff
@@ -67,7 +68,8 @@
               setIsCreateFormOpen(false);
 
               if (createdUser && password) {
-                navigate(`/users/${createdUser.id}?password=${password}`);
+                sessionStorage.setItem(generatedPasswordStorageKey, password);
```

Comment:
try to confirm: the page will be automatically directed to the user detail page, and the password stored in session storage will be retrieved and deleted asap to avoid potential security risks.

Reply:
yes